pkg/config/config.go

```diff
@@ -558,9 +558,12 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("dogstatsd_mem_based_rate_limiter.soft_limit_freeos_check.max", 0.1)
 	config.BindEnvAndSetDefault("dogstatsd_mem_based_rate_limiter.soft_limit_freeos_check.factor", 1.5)
 
-	config.BindEnvAndSetDefault("dogstatsd_context_limiter.limit", 0) // 0 = disabled.
+	config.BindEnvAndSetDefault("dogstatsd_context_limiter.limit", 0)         // 0 = disabled.
+	config.BindEnvAndSetDefault("dogstatsd_context_limiter.entry_timeout", 1) // number of flush intervals
 	config.BindEnvAndSetDefault("dogstatsd_context_limiter.key_tag_name", "pod_name")
 	config.BindEnvAndSetDefault("dogstatsd_context_limiter.telemetry_tag_names", []string{})
+	config.BindEnvAndSetDefault("dogstatsd_context_limiter.bytes_per_context", 1500)
```

Comment:
This value represents the estimated bytes used for a single context? Can we measure this? Or did this value come from some measurements we've done?

Reply:
Had the same thought during my review.

Reply:
Exactly, the number is the expected number of bytes used by a single metric context including tags and metric data. Measuring precisely is tricky as we have data spread all around multiple data structures, but it can be estimated by taking a heap snapshot and tallying up all things related to context storage and then dividing by number of contexts tracked. Naturally, there will be a lot of variance (sketches use more memory than counters and gauges, apps with lots of tags vs little, and so on), so the default value is a rounded conservative approximation based on a few such profiles I collected. I fully expect this will need to be tweaked when the feature is put to use.